source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseWalletDialog.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+import React, { Component } from 'react';
+import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
+import { Select } from 'react-polymorph/lib/components/Select';
+import { SelectSkin } from 'react-polymorph/lib/skins/simple/SelectSkin';
+import { Stepper } from 'react-polymorph/lib/components/Stepper';
+import { StepperSkin } from 'react-polymorph/lib/skins/simple/StepperSkin';
+import styles from './DelegationStepsChooseWalletDialog.scss';
+import DialogCloseButton from '../../widgets/DialogCloseButton';
+import DialogBackButton from '../../widgets/DialogBackButton';
+import Dialog from '../../widgets/Dialog';
+
+const messages = defineMessages({
+  title: {
+    id: 'delegation.setup.steps.dialog.title',
+    defaultMessage: '!!!Delegation Setup',
+    description:
+      'Title "Delegation Setup" on the delegation setup "choose wallet" step dialog.',
+  },
+  description: {
+    id: 'delegation.setup.chooseWallet.step.dialog.description',
+    defaultMessage:
+      '!!!Choose a wallet with funds you would like to delegate to a stake pool. Selected wallet needs to have a minimum of <span>1 ada</span>.',
+    description:
+      'Description on the delegation setup "choose wallet" step dialog.',
+  },
+  selectWalletInputLabel: {
+    id: 'delegation.setup.chooseWallet.step.dialog.selectWalletInputLabel',
+    defaultMessage: '!!!Wallet',
+    description:
+      'Label "Wallet" for select input on the delegation setup "choose wallet" step dialog.',
+  },
+  selectWalletInputPlaceholder: {
+    id:
+      'delegation.setup.chooseWallet.step.dialog.selectWalletInputPlaceholder',
+    defaultMessage: '!!!Select Wallet',
+    description:
+      'Placeholder "Select Wallet" for select input on the delegation setup "choose wallet" step dialog.',
+  },
+  stepIndicatorLabel: {
+    id: 'delegation.setup.chooseWallet.step.dialog.stepIndicatorLabel',
+    defaultMessage: '!!!STEP 1 OF 4',
```

Comment:
Here you could have `'STEP {currentStep} OF {totalSteps}'` then, instead of `intl.formatMessage`, use:

```js
<FormattedMessage
  {...messages.stepIndicatorLabel}
  values={{ currentStep, totalSteps }}
/>
```

Reply:
@daniloprates really nice hint, thanks